src/web/app.py

```
@@ -57,153 +59,450 @@
+            logger.debug("Including Google search entry point in response")
+
+        yield f"event: citations\ndata: {json.dumps(citations_data)}\n\n"
 
         # Signal completion
         yield f"event: done\ndata: {json.dumps({'status': 'complete'})}\n\n"
 
-        logger.info(f"Query completed with {len(enriched_citations)} citations")
+        logger.debug(f"Query completed with {len(citations)} citations")
 
+    except asyncio.TimeoutError:
+        logger.error("ADK orchestration timed out")
```

Ruff (0.14.6)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)